src/ApparentHorizons/StrahlkorperInDifferentFrame.hpp

```diff
@@ -39,3 +39,25 @@ void strahlkorper_in_different_frame(
         std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
         functions_of_time,
     double time);
+
+/// \brief Transforms a Strahlkorper from SrcFrame to DestFrame, for easy maps.
+///
+/// This is a simplified version of strahlkorper_in_different_frame
+/// with the following two assumptions: First, the map leaves the
+/// center of the Strahlkorper invariant. Second, for every point on
+/// the Strahlkorper, the map leaves the angles about the center
+/// (i.e. the quantity
+/// \f$(x^i-c^i)/\sqrt{(x^0-c^0)^2+(x^1-c^1)^2+(x^2-c^2)^2}\f$)
+/// invariant. If those assumptions are not satisfied, then
+/// strahlkorper_in_different_frame_aligned will give the wrong answer
+/// and you must use strahlkorper_in_different_frame.  Those
+/// assumptions are satisfied for the shape and size maps but not for
+/// general maps.
```

Comment:
And then add a note here that this only works for SrcFrame = Grid and DestFrame = Distorted or Inertial. Just to make it clear that one of these functions can be used in the distorted frame (currently) but the other one can't.